File: ui/conflict/src/main/java/com/laioffer/clickevent/MainActivity.java
package com.laioffer.clickevent;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.util.Log;
import android.view.MotionEvent;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.laioffer.ui.conflict.R;

public class MainActivity extends AppCompatActivity {

    private Button btn_click;

    private static final String TAG = "MainActivity";

    @SuppressLint("ClickableViewAccessibility")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_click_event);

        btn_click = findViewById(R.id.btn_click);

        btn_click.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.e(TAG, "onClick");
            }
        });

        btn_click.setOnTouchListener(new View.OnTouchListener() {

            @Override
            public boolean onTouch(View v, MotionEvent event) {
                Log.e(TAG, "onTouch: "+event.getAction());

                // return false;
                return true;
            }
        });
    }
}

File: ui/basic/src/main/java/com/laioffer/MainActivity.kt
package com.laioffer

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.laioffer.material.l1_toolbar.ToolbarActivity
import com.laioffer.material.l2_drawer.DrawerLayoutActivity
import com.laioffer.ui.basic.R

class MainActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    startNewActivity(R.id.material_toolbar_button, ToolbarActivity::class.java)
    startNewActivity(R.id.material_drawerlayout_button, DrawerLayoutActivity::class.java)

  }

  private fun startNewActivity(buttonId: Int, clazz: Class<*>) {
    val button = findViewById<Button>(buttonId)
    button.setOnClickListener {
      startActivity(Intent(this@MainActivity, clazz))
    }
  }
}